platforms/python/versionsToBuild.txt

```diff
@@ -12,7 +12,8 @@
 3.9.1rc1, E3FF2839C048B25C084DEBE9B26995E310250568,
 3.9.7, E3FF2839C048B25C084DEBE9B26995E310250568,
 
-3.10.0a2, 64E628F8D684696D,
```

Comment:
Do we need to revert the changes in this file since we aren't adding or modifying Python versions, or is this a separate change?

Reply:
The sha for 3.10.0a2 isn't correct so I updated it here. And since we've already had these versions in both dev and prod, it won't add or modify the existing ones.